### PR TITLE
Enforce phpcs severity=1 in CI

### LINKED ISSUE #544: Fix PHPCS violations with --severity=1
## Describe the bug
We currently run PHPCS with the default severity of 5, but since these error and warning violations are all addressed by default, we should lower the severity to 1, and then address those items as well, and leave the severity at 1 moving forwards.

## To Reproduce

1. Run `composer cs -- --severity=1` or `php ./vendor/bin/phpcs --severity=1`.
2. See the violations, and start fixing them. Fixes may include changing the code, adding ignore comments, or adjusting the PHPCS configuration.

## Expected behaviour
After fixing, there should be no PHPCS violations at severity 1.

## Environment

Latest PHPCS, WPCS, VIPCS and related standards.

## Additional context
The severity=1 report gives:

```txt
W.E.WWW 7 / 7 (100%)



FILE: tests/Integration/RestTest.php
----------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AND 1 WARNING AFFECTING 2 LINES
----------------------------------------------------------------------------------------------------------------
 130 | WARNING | This comment is 72% valid code; is this commented out code?
     |         | (Squiz.PHP.CommentedOutCode.Found)
 130 | ERROR   | Empty line required before block comment (Squiz.Commenting.BlockComment.NoEmptyLineBefore)
 131 | ERROR   | Block comments must start with a capital letter (Squiz.Commenting.BlockComment.NoCapital)
----------------------------------------------------------------------------------------------------------------


FILE: views/json-ld.php
------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 17 | WARNING | Variable $parsely_page is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
------------------------------------------------------------------------------------------------------------------------


FILE: src/class-scripts.php
-------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------
 174 | WARNING | Not using strict comparison for in_array; supply true for third argument.
     |         | (WordPress.PHP.StrictInArray.MissingTrueStrict)
-------------------------------------------------------------------------------------------------------------------------------------------


FILE: views/repeated-metas.php
--------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
--------------------------------------------------------------------------------------------------------------------------------
 15 | WARNING | Variable $parsely_metas is undefined. (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
 25 | WARNING | Variable $parsely_page_authors is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
--------------------------------------------------------------------------------------------------------------------------------


FILE: views/custom-metadata.php
------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 16 | WARNING | Variable $parsely_page is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
------------------------------------------------------------------------------------------------------------------------


FILE: tests/Integration/ScriptsTest.php
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 6 WARNINGS AFFECTING 6 LINES
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
 128 | WARNING | Scripts should be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation.
     |         | (WordPress.WP.EnqueuedResources.NonEnqueuedScript)
 162 | WARNING | Scripts should be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation.
     |         | (WordPress.WP.EnqueuedResources.NonEnqueuedScript)
 259 | WARNING | Scripts should be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation.
     |         | (WordPress.WP.EnqueuedResources.NonEnqueuedScript)
 348 | WARNING | switch_to_blog() may not work as expected since it only changes the database context for the blog and does not load the plugins or theme of that
     |         | site. Filters or hooks on the blog you are switching to will not run.
     |         | (WordPressVIPMinimum.Functions.RestrictedFunctions.switch_to_blog_switch_to_blog)
 390 | WARNING | switch_to_blog() may not work as expected since it only changes the database context for the blog and does not load the plugins or theme of that
     |         | site. Filters or hooks on the blog you are switching to will not run.
     |         | (WordPressVIPMinimum.Functions.RestrictedFunctions.switch_to_blog_switch_to_blog)
 416 | WARNING | Scripts should be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation.
     |         | (WordPress.WP.EnqueuedResources.NonEnqueuedScript)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------


FILE: src/class-parsely.php
-------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
-------------------------------------------------------------------------------------------------------------------------------------------
 410 | WARNING | Not using strict comparison for in_array; supply true for third argument.
     |         | (WordPress.PHP.StrictInArray.MissingTrueStrict)
 986 | WARNING | Not using strict comparison for in_array; supply true for third argument.
     |         | (WordPress.PHP.StrictInArray.MissingTrueStrict)
-------------------------------------------------------------------------------------------------------------------------------------------


FILE: src/UI/class-settings-page.php
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
 259 | WARNING | urlencode() should only be used when dealing with legacy applications rawurlencode() should now be used instead. See
     |         | http://php.net/manual/en/function.rawurlencode.php and http://www.faqs.org/rfcs/rfc3986.html
     |         | (WordPress.PHP.DiscouragedPHPFunctions.urlencode_urlencode)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 7.85 secs; Memory: 6MB

### PR DESCRIPTION
**Pull request by jblz**
## Description

* Add the `--severity=1` flag to the calls to `phpcs` in:
  * the `cs-lint.yml` github action workflow
  * the `composer cs` script
* Remove the `--severity=1` flag from the `composer cs` call in the pre-commit hook now that it's included in the underlying composer script. 

Fixes #544

## Motivation and Context

Now that we are enforcing coding standards at commit time with "severity=1" and the codebase is compliant (per #538), this change also extends the granularity to the check performed in CI and via manual runs on the CLI.

## How Has This Been Tested?

* Bypassed the pre-commit hook to commit a warning violation
  * https://github.com/Parsely/wp-parsely/pull/549/commits/32b16f847b2a8a1ac07b9aedb0ec697726c384b4
  * Confirmed the change to the CI script [caught the error](https://github.com/Parsely/wp-parsely/runs/4551289531?check_suite_focus=true)
* Reverted the demo
  * (re) confirmed that tests now pass
* Confirm that pre-commit hooks still apply the coding standards w/ the `severity=1` flag included

### CODE REVIEW

**Review by sjinks — DISMISSED** [review]

**Review by GaryJones — CHANGES_REQUESTED** [review]
This should be changed in composer scripts as well so we test the same thing locally and also on pre-commit.

File: .github/workflows/cs-lint.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
 
       # Check the code-style consistency of the PHP files.
       - name: Check PHP code style
-        run: vendor/bin/phpcs --report-full --report-checkstyle=./phpcs-report.xml
+        run: vendor/bin/phpcs --report-full --report-checkstyle=./phpcs-report.xml --severity=1
```
Comment (jblz) [inline .github/workflows/cs-lint.yml]: Any reason we shouldn't go ahead and change this to:
```suggestion
        run: composer cs -- --report-full --report-checkstyle=./phpcs-report.xml
```
...?

Comment (GaryJones) [reply .github/workflows/cs-lint.yml]: As long as it still works, I have no objections.

**Review by GaryJones — DISMISSED** [review]

**Review by GaryJones — APPROVED** [review]
✅

**Pull request merged**